### PR TITLE
DietPi-Environment | Some service-, config- and code adjustments

### PR DESCRIPTION
**Pull request by MichaIng**
+ DietPi-Environment | Move sudo adjustments to "/etc/sudoers.d/dietpi"
+ DietPi-Environment | Leave rc-local services unchanged from now on, prevent incompatibilities by DietPi-PostBoot by "Before=rc-local.service rc.local.service"
+ DietPi-Environment | Move sysctl adjustments to "/etc/sysctl.d/dietpi.conf" to assure higher priority than "/etc/sysctl.d/99-sysctl.conf"
+ DietPi-Set_core_environment/Patchfile/Set_Software | Minor code enhencements
+ DietPi-Patchfile | Avoid doubled running of DietPi-Set_core_environment
+ DietPi-Set_Software | "verify_dietpi.txt": Reduce output to only mention added entries

**Whoopsie: Accidentally added changelog entries to testing branch 😅: https://github.com/Fourdee/DietPi/commit/045701ad3162a858676bb9e144bd3a42c0d4ed31**

### CODE REVIEW

File: dietpi/func/dietpi-set_core_environment [hunk 1]
```diff
@@ -279,60 +279,14 @@ exit 0
 _EOF_
 	chmod +x /var/lib/dietpi/dietpi-software/services/kill-ssh-user-sessions-before-network.sh
 
-	G_DIETPI-NOTIFY 2 'Configuring rc-local service:'
```
Comment (Fourdee) [inline dietpi/func/dietpi-set_core_environment:282]: This needs to be added to PREP?

Comment (Fourdee) [reply dietpi/func/dietpi-set_core_environment:282]: Or we could: keep the service creation in core-env, then manually create ```/etc/rc.local``` outside of it?

Comment (MichaIng) [reply dietpi/func/dietpi-set_core_environment:282]: Do we want to assure rc.local functionality with DietPi? By default it is implemented moreless, if I remember right rc.local file itself is not present on all Debian images by default, but the services are in place.

I thought about not touching this on new images, leaving it to user if needed and just assuring that it runs after DietPi-PostBoot (Before=).

Comment (Fourdee) [reply dietpi/func/dietpi-set_core_environment:282]: @MichaIng 

> I thought about not touching this on new images, leaving it to user if needed and just assuring that it runs after DietPi-PostBoot (Before=).

Agree, postboot is a better feature/system 👍  Lets go with that.

**Comment by Fourdee** [comment]
@MichaIng 

Legend, thank you 👍

**Pull request merged**